WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderDetailsViewModel.swift

```diff
@@ -201,6 +201,12 @@ extension OrderDetailsViewModel {
         case .details:
             ServiceLocator.analytics.track(.orderDetailProductDetailTapped)
             viewController.performSegue(withIdentifier: Constants.productDetailsSegue, sender: nil)
+        case .refund:
+            ServiceLocator.analytics.track(.orderDetailRefundDetailTapped)
+            let index = indexPath.row - Constants.paymentSummaryCell - Constants.paidByCustomerCell
```

Comment:
from looking into the responsibility of `OrderDetailsViewModel` vs. `OrderDetailsDataSource`, it seems like `OrderDetailsDataSource` owns the implementation details of how the table view cells are configured (like the off by 2 index). How about abstracting this detail from `OrderDetailsViewModel` by moving the code above and below to `OrderDetailsDataSource` into a function like `func refund(at indexPath: IndexPath) -> Refund` so that the view model doesn't have to keep the cell offset constants in sync with the data source?

Reply:
ah, yeah that makes sense. +1